managed/src/main/resources/db/migration/default/common/V64__Alter_Customer_Config_Table.sql

```diff
@@ -0,0 +1,4 @@
+-- Copyright (c) YugaByte, Inc.
+
+alter table if exists customer_config add column if not exists config_name varchar(64) null;
+update customer_config set config_name = concat(name, '-Default');
```

Comment:
Migration procedures should be idempotent. Here it is not (the second operation). If the table already has column `config_name`,  it resets the column content.

Reply:
yeah, I have updated. Is it fine now? I am checking for null for the column name.